### DISCUSSION TITLE
Unit DotPlot

### DISCUSSION THREAD

**Discussion by MDMLD** [satijalab/seurat#8842] [General]
Hey guys, I am currently using Seurat v.4.3.0.1 and I was wondering, what unit does the average expression in the DotPlots have? I thought they re simply UMI, but that doesn´t fit with my data? Would be happy if somebody could help me with that issue!

**Comment by samuel-marsh** [comment]
The default is to plot scaled data for the DotPlot. 

Best,
Sam
